include/sound/sof/ext_manifest.h

```diff
@@ -61,8 +61,20 @@ enum sof_ext_man_elem_type {
 	SOF_EXT_MAN_ELEM_WINDOW			= SOF_IPC_EXT_WINDOW,
 	SOF_EXT_MAN_ELEM_CC_VERSION		= SOF_IPC_EXT_CC_INFO,
 	SOF_EXT_MAN_ELEM_DBG_ABI		= SOF_IPC_EXT_USER_ABI_INFO,
+	SOF_EXT_MAN_ELEM_CONFIG_DATA		= 5,
```

Comment:
This actually seems error prone to me. Imagine someone adding `#define SOF_IPC_EXT_MY_NEW_THING 5` and then adding `SOF_EXT_MAN_ELEM_SAME_NEW_THING = SOF_IPC_EXT_MY_NEW_THING` here and you get two enum macros with the same value. This should be replaced with explicit values
```
enum sof_ext_man_elem_type {
	SOF_EXT_MAN_ELEM_FW_VERSION		= 0,
	SOF_EXT_MAN_ELEM_WINDOW			= 1,
	SOF_EXT_MAN_ELEM_CC_VERSION		= 2,
	SOF_EXT_MAN_ELEM_DBG_ABI		= 4,
};
```
Unrelated to this patch

Reply:
You're right. SOF_IPC_EXT* need to be clean up after we switch to extended manifest.
I will submit enhancement/clean up patch after this PR.